### ISSUE TITLE
Automated builds of the Visual Studio extension

### ISSUE THREAD

**Issue by Metadorius** [stride3d/stride#1292]
**Is your feature request related to a problem? Please describe.**
Currently Visual Studio extension only supports VS2019. The changes to support VS2022 are in the repo already but weren't released yet. In order to get the latest version I need to clone the whole Stride repo and build it myself, which is not convenient.

**Describe the solution you'd like**
A GitHub Actions (or other automation platform) workflow to compile the extension automatically on push. The resulting vsix would be then available via run artifacts. [Here's](https://github.com/Phobos-developers/Phobos/blob/develop/.github/workflows/nightly.yml) how I set it up for my project, also see [nightly.link](https://nightly.link) for convenient artifact downloads.

**Describe alternatives you've considered**
- Manually compiling the stuff: inconvenient because I need to check out quite a big repo, install dependencies etc.

**Comment by xen2** [comment]
Automatic build is happening there:
https://teamcity.stride3d.net/?guest=1

However currently the VS Package is not build automatically (https://teamcity.stride3d.net/buildConfiguration/Engine_Package_BuildPackage?mode=builds&guest=1)

**Comment by Metadorius** [comment]
Any solution which would allow people to download "nightly" builds would be nice, although I am in favor of no-signup solutions. nightly.link does just that. Does TeamCity allow guests to download build artifacts?

**Comment by xen2** [comment]
Yes, there's a guest login (and any link with `guest=1` would not even try to go through the login page first).
